cmd/tsconnect/pkg/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "author": "Tailscale Inc.",
+  "description": "Tailscale Connect SDK",
+  "license": "BSD-3-Clause",
+  "name": "@tailscale/connect",
+  "type": "module",
```

Comment:
This means we're only exporting ES Module versions here. Along with the fact that we're using `fetch` internally, that means this package won't be compatible with Node or with older browsers. I assume that's ok?

Reply:
Yeah, it's meant for use in the admin panel only.